### DISCUSSION TITLE
difficultés à l'installation de mercator

### DISCUSSION THREAD

**Discussion by ymeyer1** [dbarzin/mercator#222] [Q&A]
Bonjour,

Dans l'installation de mercator, je butte sur la partir composer. Apparemment, je dois déposer un fichier .json mais lequel? Est-ce le fichier composer.json compris dans la liste de code ? Et si oui, voici le message d'erreur que j'obtiens lorsque je lance la commande explicitée dans votre procédure:
![2022-10-01 14_50_15-Mercator - VMware Workstation](https://user-images.githubusercontent.com/114814006/193410694-2dc2b55d-d4f2-43e8-94db-40ce389e195a.jpg)

Je ne comprends pas non plus ce qu'il y a à réaliser dans la seconde partie 
Publish all publishable assets from vendor packages

php artisan vendor:publish --all

Pouvez-vous apporter plus de précisons svp ?

Cordialement,
ymeyer

**Comment by dbarzin** [comment]
Bonjour,
La procédure d'installation est testée automatiquement, elle fonctionne sans problèmes avec Ubuntu 22.04.
Quelle version de composer utilises-tu ? Quel version de PHP ? Sous quel OS ?
Merci,
Didier

**Reply by ymeyer1** [reply]
Bonjour Didier,
Désolé de revenir aussi tardivement vers vous. J'ai tout repris à zéro et l'installation s'opère finalement bien.
Merci,
Yannick

**Reply by dbarzin** [reply]
Merci pour le retour.
Cordialement,
Didier
